app/views/shared/_navbar.html.erb

```diff
@@ -29,13 +29,13 @@
     <% else %>
       <div class="navbar-categories">
```

Comment:
Your idea of putting the artist name instead of the contact navbar is good, you should try it !

Reply:
Héhé done ( this is not a navbar update branch :) )